Adds words to abstract definition. (#3080)
ethereum/EIPs: eip-template.md

If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.

## Abstract

> A short (~200 word) description of the technical issue being addressed. This should be a very terse and human readable version of the specification section.  Someone should be able to read the abstract and by the end get the gist of what this specification does.

Reviewer: ```suggestion
A short (~200 word) description of the technical issue being addressed. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
```
